src/Dockerfile

```diff
@@ -27,9 +27,8 @@ ENV GO111MODULE=on
 
 # Download protoc, protoc-gen-go, protoc-gen-go-grpc
 RUN apt-get update && apt-get install --no-install-recommends --assume-yes  protobuf-compiler \
-  && go get google.golang.org/grpc \
-  && go get google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest \
-  && go get github.com/golang/protobuf/protoc-gen-go \
+  && go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@latest \
```

Comment:
reason for that change?

Reply:
Thanks for this, but isn't the equivalent of _google.golang.org/grpc_ missing?

Reply:
Thanks for this, but isn't the equivalent of google.golang.org/grpc missing?